database_refresh/02-cronjob.yaml

```diff
@@ -0,0 +1,75 @@
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: db-refresh-job
+spec:
+  schedule: "30 10 * * 0"
+  concurrencyPolicy: "Forbid"
+  successfulJobsHistoryLimit: 5
+  failedJobsHistoryLimit: 3
+  jobTemplate:
+    spec:
+      backoffLimit: 0
+      activeDeadlineSeconds: 1200
```

Comment:
We can also add an automatic cleanup of completed jobs with this (24 hours for example):
`ttlSecondsAfterFinished: 86400`

Reply:
I don't think this is necessary anymore - successful jobs and their associated pods don't persist for anytime at all anymore.